src/main/scala/atom/Atom.scala

```diff
@@ -0,0 +1,113 @@
+package laughedelic.atom
+
+import scala.scalajs.js, js.annotation._
+
+@js.native
+trait Disposable extends js.Object {
+  def dispose(): Unit = js.native
+}
+
+@js.native
+trait CommandRegistry extends js.Object {
```

Comment:
Nice! 👍 This will be useful to add some custom metals commands (like cache reset).

Reply:
I added because https://github.com/laughedelic/atom-ide-scala/issues/27 =)